include/clang/Basic/DiagnosticSemaKinds.td

```diff
@@ -8945,6 +8945,13 @@ def err_bounds_type_annotation_lost_checking : Error<
 
   def err_checked_scope_no_variadic_func_for_expression : Error<
     "cannot use a variable arguments function in a checked scope or function">;
+
+  def err_bounds_cast_error_with_single_syntax
+      : Error<"invalid bounds cast: expected _Ptr or * type">;
+
+  def err_bounds_cast_error_with_array_syntax
+      : Error<"invalid bounds cast: expected Array_ptr type">;
```

Comment:
There's a type: the underscore in front of the type name is missing. It should `_Array_ptr`, not `Array_ptr`.